green_cli/utils.py

```diff
@@ -25,3 +25,10 @@ def add_utxos_to_transaction(session, details):
         utxos = gdk_resolve(gdk.get_unspent_outputs(session.session_obj, json.dumps(utxo_details)))
         details['utxos'] = utxos['unspent_outputs']
 
+def get_txhash_with_sync(session, details, wait):
+    txhash = details['txhash']
+    while wait:
```

Comment:
Do we want some kind of timeout mechanism?

Reply:
I guess we probably should, I'll add one ta

Reply:
Added `--timeout` everywhere cheers